Sources/Plasma/CoreLib/hsTemplates.h

```diff
@@ -148,6 +148,13 @@ template <class T> class hsTArray : public hsTArrayBase
     int     Count() const { return fUseCount; }
     int     GetCount() const { return fUseCount; }
     inline void SetCount(int count);
+
+    void Resize(int count)
+    {
+        // WARNING: By design (sigh), the new elements are not (re)initialized...
```

Comment:
I think this should be moved up above the method as a doxygen style comment so it will pop up as a tip in the IDE.